raw/mock_exam/docs/questions/question-39.adoc

```diff
@@ -0,0 +1,90 @@
+// tag::DE[]
+=== Frage 39
+**ID: Q-20-04-36**
+
+[cols="2,8,2", frame=ends, grid=rows]
+|===
+| P-Frage: 
+| Wählen Sie die **drei** am besten passenden Antworten aus
+| 1 Punkt
+|===
+
+Sie versuchen, ihre Architektur quantitativ zu untersuchen.
+Welche der folgenden Größen können Sie für Ihre Softwarearchitektur zuverlässig messen?
+Wählen Sie die drei am besten passenden Antworten aus.
+
+[cols="1a,1,10", frame=none, grid=none]
+|===
+
+| {y}
+| (a)
+| Größe der Bausteine (z.{nbsp}B. Lines-of-Code).
```

Comment:
unklar, ob {nbsp} hier greift

Reply:
Tut es. 🙂  Habe ich getestet.

Reply:
Ah, wobei. Kann sein, dass ich ein `a` brauche, ich schaue es nochmal an.

Reply:
added a| to table row (DE-version)